GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -501,7 +501,46 @@ public GVRSceneObject getAssimpModel(String assetRelativeFilename)
      */
     public GVRSceneObject getAssimpModel(String assetRelativeFilename,
             EnumSet<GVRImportSettings> settings) throws IOException {
-        return GVRImporter.getAssimpModel(this, assetRelativeFilename, settings);
+        return GVRImporter.getAssimpModel(this, assetRelativeFilename, settings, false);
+    }
+    
+    /**
+     * Simple, high-level method to load a scene as {@link GVRSceneObject} from
+     * 3D model.
+     * 
+     * @param fileName
+     *            A filename, present in the external storage of the device
+     * 
+     * @return A {@link GVRSceneObject} that contains the meshes with textures
+     * 
+     * @throws IOException
+     *             File does not exist or cannot be read
+     *
+     */
+    public GVRSceneObject getAssimpModelFromSDCard(String fileName)
```

Comment:
Did you consider using (and extending if necessary) the GVRAndroidResource class from GVRF? Seems like we need something like it to avoid the "fan out" GVRAndroidResource mentions. Also it is already used for textures and meshes..